tools/springboot/springboot.bzl

```diff
@@ -63,6 +63,22 @@ _depaggregator_rule = rule(
     },
 )
 
+def _appjar_locator_impl(ctx):
+    if java_common.provider in ctx.attr.app_dep:
+        output_jars = ctx.attr.app_dep[java_common.provider].runtime_output_jars
```

Comment:
so runtime_output_jars does not to have the extra jdeps artifact for kotlin libs?